lib/trace.c

```diff
@@ -154,12 +154,10 @@ M0_INTERNAL void m0_trace_fini(void)
  */
 static inline uint64_t m0_rdtsc(void)
 {
-	uint32_t count_hi;
-	uint32_t count_lo;
+	uint64_t cycle;
```

Comment:
Can we use:
```
#ifdef CONFIG_X86_64
existing x86 code;
#else
new arm code;
#endif
```

Reply:
Modified the code as per your suggestion retaining both code for x86 and aarch64 platform with conditional preprocessor defines